dima/react-movies/src/pages/mainpage/index.ts

```diff
@@ -0,0 +1,21 @@
+// import { StateInterface } from "../../interfaces/stateInterface";
+import MainPage from "./mainpage";
+// import { fetchMovies, setMovies, resetMovies } from "../../redux/moviesActions";
+// import { connect, ConnectedProps } from "react-redux";
+
+// const mapStateToProps = (state: StateInterface) => {
+//   return {
+//     movies: state.movies,
+//     loading: state.loadingMovies,
+//   };
+// };
+
+// const mapDispatchToProps = {
+//   fetchMovies,
+//   setMovies,
+//   resetMovies,
+// };
+// const connector = connect(mapStateToProps, mapDispatchToProps);
+// export type MainConnectProps = ConnectedProps<typeof connector>;
+// export default connector(MainPage);
```

Comment:
I  understand that you might've left it here for study purpose. But never leave things like that on a real project!